src/Formatter/IntlMoneyFormatter.php

```diff
@@ -16,13 +16,19 @@
      * @var \NumberFormatter
      */
     private $formatter;
+    /**
+     * @var int|null
```

Comment:
`public function __construct(\NumberFormatter $formatter, $subunits = 0)` ?

Subunit is still optional